niji/main.js

```diff
@@ -0,0 +1,68 @@
+var express = require('express');
+var path = require('path');
+var favicon = require('serve-favicon');
+var logger = require('morgan');
+var bodyParser = require('body-parser');
+var cookieParser = require('cookie-parser');
+
+// var appRoutes = require('./routes/app');
+var apiRoutes = require('./routes/api');
+
+var app = express();
+
+// view engine setup
+app.set('views', path.join(__dirname, 'app/prod/hbs'));
```

Comment:
We don't have any views yet, so get rid of hbs. I doubt we'll even need a template engine since we're doing Angular.